### PR TITLE
RIA-3634 add advanced bundling

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3634

### Change description ###
see ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ccd/Event.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
     CREATE_CASE_SUMMARY("createCaseSummary"),
     REVERT_STATE_TO_AWAITING_RESPONDENT_EVIDENCE("revertStateToAwaitingRespondentEvidence"),
     GENERATE_HEARING_BUNDLE("generateHearingBundle"),
+    CUSTOMISE_HEARING_BUNDLE("customiseHearingBundle"),
+    ASYNC_STITCHING_COMPLETE("asyncStitchingComplete"),
```
Comment (makowalski) [inline src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ccd/Event.java:30]: this event is not needed but it doesn't cause issues if we keep it here

**Review by makowalski — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ccd/Event.java (hunk 1 above)
Comment (mkayad) [reply src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ccd/Event.java]: event removed

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**